package Global;

message ServerInfo
{
  required string server_address = 1;
  required uint64 server_port = 2;
}

message ServerConfig
{
  required uint64 server_id = 1;
  repeated ServerInfo server_info = 2;
}
